packages/core/src/lib/toast/toast-item.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+import type { Readable } from 'svelte/store';
+import { ToastBanner, type ToastVariantType } from '$lib/toast';
+
+export let message: string;
+export let closeable: boolean | undefined = false;
+export let variant: ToastVariantType;
+export let progress: Readable<number>;
+export let dismiss: () => unknown;
+</script>
+
+<ToastBanner
+  {closeable}
+  cx="pointer-events-auto relative mb-2 w-screen sm:w-[360px]"
+  on:close={dismiss}
+  on:action={dismiss}
```

Comment:
We _do_ need actions to work for R2D2, so this looks like an issue

Reply:
I will modify this. Question: does the success toast we're using use actions in any way? From my understanding, only the close button is displayed on a success toast. I'm asking because I might make the action field optional

Reply:
Yes, various success toasts for destructive actions include an "Undo" action